src/test/regress/specs/isolation_create_restore_point.spec

```diff
@@ -64,26 +85,39 @@ step "s1-commit"
 
 session "s2"
 
+step "s2-begin"
+{
+	BEGIN;
+}
+
 step "s2-create-restore"
 {
 	SELECT 1 FROM citus_create_restore_point('citus-test');
 }
 
+step "s2-commit"
+{
+	COMMIT;
+}
+
 # verify that citus_create_restore_point is blocked by concurrent create_distributed_table
 permutation "s1-begin" "s1-create-distributed" "s2-create-restore" "s1-commit"
 
-# verify that citus_create_restore_point is blocked by concurrent INSERT
+# verify that citus_create_restore_point is not blocked by concurrent INSERT
 permutation "s1-begin" "s1-insert" "s2-create-restore" "s1-commit"
 
-# verify that citus_create_restore_point is blocked by concurrent master_modify_multiple_shards
+# verify that citus_create_restore_point is not blocked by concurrent master_modify_multiple_shards
```

Comment:
It took me a while to get why the `create-restore` is not blocked `modify-multiple`, 2PC transactions are not blocked until `COMMIT`. Maybe a note to the header of `citus_create_restore_point` function mentioning this would be useful for future readers.